src/modules/dashboard/components/ColonyTotalFunds/ColonyTotalFunds.tsx

```diff
@@ -45,12 +45,13 @@ const ColonyTotalFunds = ({
     colonyName,
     tokens: colonyTokens,
     nativeTokenAddress,
+    version,
   },
 }: Props) => {
   const [currentTokenAddress, setCurrentTokenAddress] = useState<Address>(
     nativeTokenAddress,
   );
-
+  const isSupportedColonyVersion = version >= 5;
```

Comment:
Same here, please use the version constant from colonyJS

```suggestion
  const isSupportedColonyVersion = colony.version >= ColonyVersion.CeruleanLightweightSpaceship);
```